projects/packages/assets/composer.json

```diff
@@ -5,7 +5,9 @@
 	"license": "GPL-2.0-or-later",
 	"require": {
 		"php": ">=7.2",
-		"automattic/jetpack-constants": "@dev"
+		"automattic/jetpack-connection": "@dev",
+		"automattic/jetpack-constants": "@dev",
+		"automattic/jetpack-status": "@dev"
```

Comment:
These changes makes Phan (our static analysis tool) happy.

Reply:
That's indeed to be expected, since you're starting to rely on the Status package in the Assets package by using some of its methods.